api/src/room_store.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+
+
+class RoomStore:
+    def __init__(self):
+        self.path = os.environ['ROOM_STORE_DIR']
+        if not os.path.exists(self.path):
+            os.mkdir(self.path)
+
+    def get_all_room_ids(self) -> list:
+        return os.listdir(self.path)
+
+    def write_room_data(self, room: str, data: dict):
```

Comment:
Everywhere else this is called `room_id`, probably makes sense to continue that pattern here

Reply:
I guess